### ISSUE TITLE
Influence TX (State legislative campaign finance)

### ISSUE THREAD

**Issue by mscarey** [open-austin/project-ideas#108]
### What problem are we trying to solve?

Informing the Texas electorate about the effect of money on the legislative process.

Who has the time to search for hours and hours through newspapers and online articles in the (possibly vain) hope of finding unbiased information on each and every candidate?

By putting these campaign finance records and voting records together in a succinct mobile platform, we can turn uninformed voters into informed voters.

### Who will benefit (directly and indirectly) from this project?

Concerned voters and journalists.

### Where can we find any research/data available/articles?

Our [project repository](https://github.com/open-austin/influence-texas), including the [data folder](https://github.com/open-austin/influence-texas/tree/master/data). 

### What help is needed at this time?

Data analysis, Django, front end.

### What are the next steps (validation, research, coding, design)?

We still need more data analysis to create new informative content for users. Development of the website is going on and still needs work.

----

#### Checklist for NEW ideas :baby:
Hey, you're official! You're now part of the growing civic hacking community in Austin. Here's a few things to get started (a couple you've probably already done).

- [x] Create this idea issue
- [x] Flesh out the who, where, and what questions above
- [x] Start the conversation about this new idea on [Slack](https://openaustin-slackin.herokuapp.com/)!
- [ ] Respond with at least one update on this issue within the next month

#### Checklist for ACTIVE projects :fire:
Let's get this project started! When this idea starts taking off, the Projects Core Team will start helping this project's lead(s) out with project management and connecting you to resources you may need. To get there, please complete and check off the following:

- [ ] Complete [Open Leadership 101](https://mozilla.teachable.com/p/open-leadership-101)
- [x] Create a [GitHub repository](https://github.com/new) and Slack channel for work.
- [x] Create a [README file](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/write-a-great-project-readme/) in your project repository. This file should help newcomers understand what your project is, why it's important, and kinds of help you're looking for.
- [x] [Create issues](https://mozilla.github.io/open-leadership-training-series/articles/get-your-project-online/project-set-up-for-collaboration-with-github/#assignment--add-your-first-issue) to describe each task that you plan to do or need help with and how a contributor can get started on that task. You might start and stop a lot, so consider issues as your to-do list.

#### Checklist for FEATURED Projects :tada:
To have your project FEATURED on [Open-Austin.org](https://open-austin.org/), complete the following documentation. In past projects, well-documented featured projects have more contributions than other projects.

* [ ] In your README, link to the [Open Austin Community Participation Guidelines](https://docs.google.com/document/d/1OujyBccPpepUXSY5_nP3alzvd81WlCKSeSf_-sU5K-U/edit?usp=sharing) or write your own code of conduct.
* [ ] [Create file: LICENSE](http://choosealicense.com/) to give your project an open license, allowing for sharing and remixing.
* [ ] [Create file: CONTRIBUTING.md](https://github.com/acabunoc/mozsprint-repo-template/blob/master/CONTRIBUTING.md) so others know [how they can contribute](https://mozilla.github.io/open-leadership-training-series/articles/building-communities-of-contributors/write-contributor-guidelines/).

Once all of the above is complete,
- [ ] [Create an issue](https://github.com/open-austin/open-austin.github.io/issues/new) on the open-austin.github.io repo with the title `Add [this project] to projects page. An Open Austin leader will review this issue and post your project :balloon:

If you get stuck at any point, feel free to reach out to the leadership team on Slack by adding @leadership to your message. We're here to help you make real changes to our city.

**Comment by werdnanoslen** [comment]
Hi y'all, this project has been super active at our hack nights and I'm super thrilled. Could you post what you're working on right now? It'd definitely help with attracting other folks that don't come to hack nights (at all or as often).

**Comment by werdnanoslen** [comment]
@2echoi check out this project! You can meet us next weekend to learn more about what UX needs this team has https://www.meetup.com/Open-Austin/events/243579932/

**Comment by VictoriaODell** [comment]
Work was done at the 10/9 civic hack Saturday: 
- [Notes from CHN on where the team left off](https://docs.google.com/document/d/1TvuZmBd1QJYKo8ksiOlfXy3Knlk7jxbVuzI0itplpkg/edit?usp=sharing)
- [Issue Milestones](https://github.com/open-austin/influence-texas/milestone/1)

**Comment by lazarus1331** [comment]
Meetup on 11/3/17:
* Created KanBan board: https://waffle.io/open-austin/influence-texas
* Updated and Closed Milestone for Mozfest
* Planned for future work.

What we Need:
* We need mobile application developers to start on the frontend to consume the API @tonysyu built

**Comment by lazarus1331** [comment]
Meetup on 1/6/18:
* Merged code for improvements to web view
* Imported data into the integration live server

What we need:
* We need mobile developers to wrap the django application for delivery to different platforms.

**Comment by jplaut** [comment]
Hey all,

Not sure if this project is still active given the date of the last comment, but wanted to chime in. I'm working on a project now that correlates federal representatives' votes on bills with voters' own opinions on those bills in order to show voters their alignment with their representatives. The goal is to help voters make informed voting decisions and possibly inspire political action. I've been thinking about adding a concept of campaign contributions as well, which lines up well with what you're working on. I have an account with Open Secrets and have access to their API, which includes info on campaign contributions per rep by industry and company/entity. Would be great to find some cool ways to visualize that data (I'm a huge mapping/geo data nerd) and/or integrate it into the existing project I'm working on (https://vote.foundation). I'll be at the hack day tomorrow so hope to meet some of you there. Let me know if you think this idea fits at all with what you were working on.

**Comment by mscarey** [comment]
still very active, in case anyone is wondering. [Repo](https://github.com/open-austin/influence-texas)

**Comment by lazarus1331** [comment]
April 16, 2019 Community Action Night:

- Reviewed UI design updates from Virgil
- Worked on Tests [110](https://github.com/open-austin/influence-texas/issues/110)
- Discussed setting up a Discourse forum with Open-Austin Leadership
